x-pack/plugins/security/server/routes/session_management/invalidate.ts

```diff
@@ -37,14 +37,19 @@ export function defineInvalidateSessionsRoutes({
           ),
         }),
       },
+      security: {
+        authz: {
+          requiredPrivileges: ['sessionManagement'],
```

Comment:
__question:__ just for my understanding, I see we renamed `access:manageSpaces` to `manage_spaces`, but `access:sessionManagement` stayed as `sessionManagement`, is it intentional?

Reply:
Probably, we just forgot to rename it too, there should consistency between naming

Reply:
Created an issue for that https://github.com/elastic/kibana/issues/198716